character/create.html

```diff
@@ -125,6 +125,172 @@ <h2>General Information</h2>
                             <input type="text" class="form-control" name="other" id="other">
                         </div>
                     </div>
+
+                    <!-- Someone who isn't terrible at HTML would know if inserting empty paragrah tags -->
+                    <!-- To add vertical spacing is a good idea or not. I just know it works            -->

```

Comment:
<br> my friend :)

Reply:
It's generally frowned upon for accessibility reasons- someone using a screenreader would hear that a paragraph has begun and not hear the content, and not know if the paragraph is empty or broken or what